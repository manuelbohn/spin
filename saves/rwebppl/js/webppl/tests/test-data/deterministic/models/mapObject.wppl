mapObject(
    function(key, val) {return val + 1;},
    {'a': 1, 'b': 2, 'c': 7}
);
