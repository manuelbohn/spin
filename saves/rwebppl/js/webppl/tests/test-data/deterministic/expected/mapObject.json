{
  "result": {"a": 2, "b": 3, "c": 8}
}
